tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/String/StringConcatTests.cs

```diff
@@ -265,7 +265,7 @@ public void GivenATaintedString_WhenCallingConcatWithStringNullParam_ResultIsTai
     [Fact]
     public void GivenATaintedString_WhenCallingConcat_ResultIsTainted()
     {
-        AssertTaintedFormatWithOriginalCallCheck(":+-tainted-+:", String.Concat(taintedValue), () => String.Concat(taintedValue));
+        AssertTaintedFormatWithOriginalCallCheck(":+-tainted-+:", String.Concat(taintedValue), () => String.Concat(new[] { taintedValue }));
```

Comment:
Is this change required?

Reply:
Unfortunately, yes 😢 With the new overload rules in C#13, this selects the `Concat(ReadOnlySpan<char>)` overload. And then the compiler fails to compile it, because you can't have `ReadOnlySpan<>` in an `Expression` (or something like that). That was the cause of all the changes in this commit: https://github.com/DataDog/dd-trace-dotnet/pull/6265/commits/4b7dd3f8daaecf5a76b9070ed3e18de535f69222